CHANGELOG.md

```diff
@@ -17,6 +17,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 -   Only running React 16 tests on CircleCI (when an explicit envvar is set)
 -   Adds an example app with Email Verification with OTP
 
+### Changes
+
+-   Clearing errors when switching between sign-in and up
+
```

Comment:
Probably should be in unreleased section

Reply:
Huh, it was there, just the merge messed it up.